controllers/keystoneapi_controller.go

```diff
@@ -1203,6 +1216,35 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 			endptConfig["SSLCertificateFile"] = fmt.Sprintf("/etc/pki/tls/certs/%s.crt", endpt.String())
 			endptConfig["SSLCertificateKeyFile"] = fmt.Sprintf("/etc/pki/tls/private/%s.key", endpt.String())
 		}
+
+		endptConfig["EnableFederation"] = false // default OIDCFederation to false, and set it below to true if enabled
+		if instance.Spec.EnableFederation {
+			ospSecret, _, err := oko_secret.GetSecret(
+				ctx,
+				h,
+				instance.Spec.Secret,
+				instance.Namespace)
+			if err != nil {
+				return err
+			}
+
```

Comment:
Now that you have the ospsecret, you might want to validate that the contents include the KeystoneOIDCClientSecret and KeystoneOIDCCryptoPassphrase and message and error out appropriately.  Right now this will either throw an error or worse put a blank string into the config.

Reply:
Good call, I'll fix that.

Reply:
Done